src/test/java/teammates/test/cases/webapi/BaseActionTest.java

```diff
@@ -177,6 +180,23 @@ protected void loginAsStudentInstructor(String userId) {
         assertFalse(user.isAdmin);
     }
 
+    protected void updateInstructorWithOnlySectionPrivilege(InstructorAttributes instructor,
+                                                            String privilege, String[] sections, String[] submissionParams) {
+        InstructorPrivileges instructorPrivileges = new InstructorPrivileges();
+
+        for (String section : sections) {
+            instructorPrivileges.updatePrivilege(section, privilege, true);
+        }
+
+        try {
+            logic.updateInstructor(
+                    InstructorAttributes.updateOptionsWithEmailBuilder(instructor.getCourseId(), instructor.email)
+                            .withPrivileges(instructorPrivileges).build());
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
We should just throw the exception instead of keep it silent? Same for other places.